From: Simon McVittie <smcv@debian.org>
Date: Fri, 26 Aug 2022 13:50:51 +0100
Subject: docs: Remove external image references

These can be considered to be a privacy breach when included in offline
documentation.

Forwarded: not-needed, Debian-specific
---
 docs/index.rst | 30 +++---------------------------
 1 file changed, 3 insertions(+), 27 deletions(-)

diff --git a/docs/index.rst b/docs/index.rst
index d729893..86e85f0 100644
--- a/docs/index.rst
+++ b/docs/index.rst
@@ -3,28 +3,9 @@
 .. SPDX-License-Identifier: MIT
 
 
-*************************
-meson-python |PyPI badge|
-*************************
-
-.. image:: https://results.pre-commit.ci/badge/github/mesonbuild/meson-python/main.svg
-   :target: https://results.pre-commit.ci/latest/github/mesonbuild/meson-python/main
-   :alt: pre-commit.ci status
-
-
-.. image:: https://github.com/mesonbuild/meson-python/actions/workflows/tests.yml/badge.svg
-   :target: https://github.com/mesonbuild/meson-python/actions/workflows/tests.yml
-   :alt: Github Action 'tests' workflow status
-
-
-.. image:: https://codecov.io/gh/mesonbuild/meson-python/branch/main/graph/badge.svg?token=xcb2u2YvVk
-   :target: https://codecov.io/gh/mesonbuild/meson-python
-   :alt: Codecov coverage status
-
-
-.. image:: https://readthedocs.org/projects/meson-python/badge/?version=stable
-   :target: https://meson-python.readthedocs.io/en/stable/?badge=stable
-   :alt: Documentation Status
+************
+meson-python
+************
 
 
 .. highlights::
@@ -159,11 +140,6 @@ Contributors
    Issue Tracker <https://github.com/mesonbuild/meson-python/issues>
 
 
-.. |PyPI badge| image:: https://badge.fury.io/py/meson-python.svg
-   :target: https://pypi.org/project/meson-python
-   :alt: PyPI version badge
-
-
 .. _Meson: https://github.com/mesonbuild/meson
 .. _getting started: usage/start.html
 .. _pip: https://github.com/pypa/pip
